### PR TITLE
Add Sufficient Assets section

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/learn/learn-assets.md [hunk 1]
```diff
@@ -34,6 +34,31 @@ The Asset Hub does not support smart contracts. See the [Advanced](#advanced-tec
...
 logic.
 
+## Sufficient Assets
+
+A sufficient asset allows an account to exist in the network’s state (i.e., a sufficient asset can
+keep an account alive on the network). Only the network's governance mechanism can deem an asset as
+_sufficient_. A balance of a non-sufficient asset can only exist on accounts that are on-chain
+(i.e., accounts having the
+[existential deposit](./learn-accounts.md#existential-deposit-and-reaping) of a sufficient asset).
+That is, a user could not keep an account on-chain by transferring an insufficient asset to it; the
```
Comment (profradha) [inline docs/learn/learn-assets.md]:
```suggestion
That is, a user could not keep an account on-chain by transferring an non-sufficient asset to it; the
```

File: docs/learn/learn-assets.md [hunk 2]
```diff
@@ -34,6 +34,31 @@ The Asset Hub does not support smart contracts. See the [Advanced](#advanced-tec
...
+account must already be on-chain by having more than the existential deposit in
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} (or a sufficient asset).
+
+Assets deemed _sufficient_ can instantiate accounts and pay for transaction fees, such that users
+can transact on the Asset Hub without the need for
+{{ polkadot: DOT. :polkadot }}{{ kusama: KSM. :kusama }} An example would be USDT on the Polkadot
```
Comment (profradha) [inline docs/learn/learn-assets.md]: concise

```suggestion
Assets deemed _sufficient_ can instantiate accounts on the Asset Hub and pay for transaction fees without the need for
{{ polkadot: DOT. :polkadot }}{{ kusama: KSM. :kusama }} An example would be USDT on the Polkadot
```

File: docs/learn/learn-assets.md [hunk 3]
```diff
@@ -34,6 +34,31 @@ The Asset Hub does not support smart contracts. See the [Advanced](#advanced-tec
 the bottom for a discussion on using proxy and multisig accounts to replicate oft-used contract
 logic.
 
+## Sufficient Assets
+
+A sufficient asset allows an account to exist in the network’s state (i.e., a sufficient asset can
+keep an account alive on the network). Only the network's governance mechanism can deem an asset as
+_sufficient_. A balance of a non-sufficient asset can only exist on accounts that are on-chain
```
Comment (profradha) [inline docs/learn/learn-assets.md]:
```suggestion

A sufficient asset on Asset Hub can allow for an account to exist on-chain even though it does not have any account balance in {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}. Any registered asset on the Asset Hub can be made _sufficient_ through governance on the Relay Chain. A balance of a non-sufficient asset can only exist on accounts that are on-chain
```

**Review by DrW3RK — APPROVED** [review]
minor suggestions. Good to go.

**Pull request merged**